features/support/visiting_pages.rb

```diff
@@ -7,3 +7,7 @@ def visit_path(path)
     visit "#{path}?smokey_cachebust=#{rand.to_s}"
   end
 end
+
+def wait_for_any_redirects_to_govuk
+  page.has_css?(".govuk-template")
```

Comment:
Could you add a quick comment here summarising what you've said in the commit description?